### PR TITLE
[Blocks editor] Images too big in large viewports

### PR DESCRIPTION
**Pull request by simotae14**
### What does it do?

Fix the dimensions and ratio of big images uploaded in the Blocks editor

### Why is it needed?

Because with big images we have too much space taken in the editor

### How to test it?

Upload images with large width or height in the Blocks editor and check the rendering

### CODE REVIEW

File: packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/useBlocksStore.js [hunk 1]
```diff
@@ -233,7 +233,9 @@ const handleEnterKeyOnList = (editor) => {
 };
 
 const Img = styled.img`
+  max-height: 512px;
```
Comment (joshuaellis) [inline packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/useBlocksStore.js:236]: this number looks arbitrary – i'm sure it's not, can you write a comment where this number comes from e.g. design

File: packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/useBlocksStore.js [hunk 2]
```diff
@@ -233,7 +233,9 @@ const handleEnterKeyOnList = (editor) => {
...
   max-width: 100%;
+  object-fit: contain;
```
Comment (madhurisandbhor) [inline packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/useBlocksStore.js:238]: When you have `object-fit:contain`, user won't understand where to begin typing for the next line, may be adding a background for image container would help, could you please confirm with UX.

File: packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/useBlocksStore.js [hunk 3]
```diff
@@ -232,20 +232,24 @@ const handleEnterKeyOnList = (editor) => {
   }
 };
 
+// The max-height is decided togher with the design team
 const Img = styled.img`
+  max-height: 512px;
```
Comment (madhurisandbhor) [inline packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/useBlocksStore.js:237]: I think this height is same as the editor height design wants to set, but now that we have toolbar also inside the editor height, this image is not fully visible. IMO We might need to subtract toolbar height.

So you can fix typo and update comment.

Comment (simotae14) [reply packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/useBlocksStore.js:237]: We wanted to respect a maximum height like we did for the Markdown Rich text editor, if you have a big screen in the future we want to provide also the possibility to expand the Editor like we did for the Markdown Rich Text Editor

**Review by Feranchz — APPROVED** [review]
I tested it locally and its working fine, great job with the images feature 👍 

I'm not sure if this is on the roadmap but it would be nice if we can add some focus styles to the image. Right now if you select an image inside the editor is not clear that your cursor is in that image.

But for the scope of this PR, everything great 🚀

**Pull request merged**